core/config/template/Zwavejs2mqtt_Fibaro_FGRGBW-442_RGBW_Controller_2.json

```diff
@@ -0,0 +1,806 @@
+{
+    "Zwavejs2mqtt Fibaro FGRGBW-442 RGBW Controller 2": {
```

Comment:
Zwavejs2mqtt has been renames [zwave-js-ui](https://github.com/zwave-js/zwave-js-ui) :-(

Reply:
Yes I saw that a few days ago. 
I'll think about renaming everything for the next beta.
A little grep/sed will certainly do the trick.